LinearApproach/LinearApproachMultiCCA.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Again, import instead please~</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/sparsecca/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>so we just use the last line than? That is obviously always possible, ll the above can be done with <strong>iterative_process_K(datasets, K)</strong>, but I thought the idea of the notebook was to break things down? This is just not possible for the iterative process tho... should we delete K&gt;1 altogether from the notebook? Or just add<strong> iterative_process_K(datasets, K)</strong> as the first cell to tell users that this function does the below for K=1?</p>

Reply:
<p>updated in notebook</p><p> </p><p> </p>